pypeit/specobjs.py

```diff
@@ -262,10 +262,14 @@ def get_std(self, multi_spec_det=None):
         pypeline = (self.PYPELINE)[0]
         if 'MultiSlit' in pypeline or 'IFU' in pypeline:
             # Have to do a loop to extract the counts for all objects
-            if self.OPT_COUNTS[0] is not None:
-                SNR = np.median(self.OPT_COUNTS*np.sqrt(self.OPT_COUNTS_IVAR), axis=1)
-            elif self.BOX_COUNTS[0] is not None:
-                SNR = np.median(self.BOX_COUNTS*np.sqrt(self.BOX_COUNTS_IVAR), axis=1)
+            if len(self.OPT_COUNTS) != 0 and self.OPT_COUNTS[0] is not None:
```

Comment:
I am not following why these changes were necessary. Can you please elaborate.

Reply:
The bug is that OPT_COUNTS is a vector of vectors, and not a matrix, so you can't use the "axis" keyword. I agree though, I'm surprised this was never encountered before... I'm not sure if this is something due to a recent PR, and not captured by the dev suite?

Perhaps in all previous reductions only a single standard star was identified? In the case of the data that I'm reducing, there are multiple stars detected in the standard star exposure. Otherwise, I'm stumped why this is something previously not encountered???

Reply:
I think your guess is a good one, i.e. multiple standards

Reply:
Hi @rcooke-ast, sorry to be picky here. Can you just use np.atleast_2d or np.reshape to cast the opt_counts to a 2d array and then use the axis keyword and forego the loop. That is less code and easier to read, but then I'm not sure I follow the shape of opt_counts as a vector of vectors so this may not work.

Reply:
Strange... I'm no longer getting multiple standards... so I just reverted the code and had no issues... Nothing obvious has changed in recent PRs, so it's a mystery to me...